samples/nrf9160/serial_lte_modem/Kconfig

```diff
@@ -16,6 +16,11 @@ config SLM_AT_MODE
 	select AT_CMD_PARSER
 	select MODEM_INFO
 
+config SLM_TEST_MODE
```

Comment:
Is `SLM_TEST_MODE` something temporary for development or will this config stay? In case of the latter, I'd add better documentation and specify what exacly does this config enable.

Reply:
It will stay and set false by default. As client, it start receiving after sending; as server, it start sending after receiving. Kind of Echo test.